services/galley/schema/src/V35.hs

```diff
@@ -0,0 +1,11 @@
+module V35 (migration) where
+
+import Imports
+import Cassandra.Schema
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 34 "Delete deprecated legalhold_team_config" $ do
```

Comment:
I'd suggest _not_ adding this migration on this PR; the main concern I have with this is: we will follow the playbook and run the migration before deploying new services and that will break the currently running service, which is why I left the comment.

I don't think it harms to make a separate PR and merge it after but I leave the decision to the PR maker

Reply:
summary of off-line discussion: generally yes, this should be spread out over two PRs, but since we promised to break staging and there is no legalhold code in prod yet, we're going to make an exception this time, and do it in one step.